android/res/layout/fragment_editor.xml

```diff
@@ -248,23 +248,32 @@
             android:src="@drawable/ic_cuisine"/>
 
           <TextView
-            android:id="@+id/tv__cuisine"
+            android:id="@+id/title_cuisine"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginLeft="@dimen/editor_margin_left"
+            android:layout_marginStart="@dimen/editor_margin_left"
+            android:text="@string/street"
+            android:textAppearance="@style/MwmTextAppearance.Body4"/>
+
+          <TextView
+            android:id="@+id/cuisine"
             style="@style/MwmWidget.Editor.FieldLayout"
-            android:layout_alignParentBottom="true"
-            android:layout_marginBottom="@dimen/margin_base"
+            android:layout_below="@id/title_cuisine"
+            android:layout_marginTop="@dimen/margin_quarter"
             android:drawableEnd="@drawable/ic_arrow_down"
             android:drawableRight="@drawable/ic_arrow_down"
             android:gravity="center_vertical"
             android:textAppearance="@style/MwmTextAppearance.Body1"
-            tools:text="Italian, russian."/>
+            tools:text="Italian, russian, russian, russian, russian, russian, russian, russian, russian"/>
```

Comment:
Кухня наша!
> On Mar 22, 2016, at 11:56 AM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/res/layout/fragment_editor.xml <https://github.com/mapsme/omim/pull/2391#discussion_r56953048>:
> 
> >              android:drawableEnd="@drawable/ic_arrow_down"
> >              android:drawableRight="@drawable/ic_arrow_down"
> >              android:gravity="center_vertical"
> >              android:textAppearance="@style/MwmTextAppearance.Body1"
> > -            tools:text="Italian, russian."/>
> > +            tools:text="Italian, russian, russian, russian, russian, russian, russian, russian, russian"/>
> Больше русских!
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2391/files/2a35d07d6ff679ab0d7c790a56d0cde70d6d95f6..f423aeb4c2632729a07c8764af99345e0b13e77b#r56953048>